testing/solomachine.go

```diff
@@ -202,7 +202,7 @@ func (solo *Solomachine) CreateHeader(newDiversifier string) *solomachine.Header
 // CreateMisbehaviour constructs testing misbehaviour for the solo machine client
 // by signing over two different data bytes at the same sequence.
 func (solo *Solomachine) CreateMisbehaviour() *solomachine.Misbehaviour {
-	merklePath := solo.GetClientStatePath("counterparty")
+	merklePath := commitmenttypes.NewMerklePath(host.FullClientStatePath("counterparty"))
 	path, err := solo.cdc.Marshal(&merklePath)
 	require.NoError(solo.t, err)
```

Comment:
we should be removing merkle path usage in solo machines

Reply:
True! I will do that!

Reply:
Is this going to be fixed in a followup?

Reply:
I cannot change this one at the moment because the we require at the moment the path to be a proto-marshalled `MerklePath` when submitting misbehaviour. I will open an issue for this.